### PR TITLE
Fix/login error redirection

### LINKED ISSUE #76: Include Code Climate support
https://codeclimate.com/changelog/1522435715584/

### PR DESCRIPTION
**Pull request by glm4**
Fixes #76 and #77 by:

-Adding CodeClimate config file with swiftlint plugin enabled.
-Adding clear instructions for automatic logout feature about redirecting the user.

## Reviewers
 
* @pMalvasio 
* @fernandatoledo
* @CamilaMoscatelli 
* @mjurfest 
* @mato2593 
* @aguscha333 
* @ximenaperez 
* @P0nj4

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#CodeClimate config file
+
+plugins:
+  swiftlint:
+    enabled: true
```
Comment (grilix) [inline .codeclimate.yml]: Can we have a newline here?

Comment (glm4) [reply .codeclimate.yml]: Sure! my bad

Comment (MaicolBen) [reply .codeclimate.yml]: isn't this the default? I don't see the purpose of this file

**Comment by MaicolBen** [comment]
@glm4 Add the CC badge to the readme

File: .codeclimate.yml (hunk 1 above)
Comment (glm4) [reply .codeclimate.yml]: @MaicolBen You say that enabling the plugin in the CC dashboard is the same as this?

Comment (MaicolBen) [reply .codeclimate.yml]: When you add the project to CC, it already runs for swiftlint, this file is for disabling or customizing the plugin checks.

Comment (glm4) [reply .codeclimate.yml]: Ok, I will delete this file and leave the Swiftlint pluging option enabled on CC.

**Pull request merged**